oscillogram_classification/cam.py

```diff
@@ -287,8 +287,11 @@ def gen_multi_chan_heatmaps_overlay_side_by_side(
     axes[0].set_ylabel("norm. voltage (V)", fontsize=18)
 
     for i in range(len(cams)):
+        voltage_val_diff = np.max(voltage_vals[i]) - np.min(voltage_vals[i])
+        voltage_val_diff = np.max(voltage_val_diff, 0.01)
```

Comment:
`TypeError: 'float' object cannot be interpreted as an integer`. The `0.01` is passed as `axis` argument.
`np.max()` expects an `array_like`, e.g., `np.max([voltage_val_diff, 0.01])`.